lib/components/user/with-logged-in-user-support.js

```diff
@@ -21,11 +20,9 @@ import UserLoaderScreen from './user-loader-screen'
  */
 const withLoggedInUserSupport = (WrappedComponent, requireLoggedInUser) =>
   props => (
-    <Auth0Wrapper showAwaitingScreen={requireLoggedInUser}>
-      <UserLoaderScreen requireLoggedInUser={requireLoggedInUser}>
-        <WrappedComponent {...props} />
-      </UserLoaderScreen>
-    </Auth0Wrapper>
+    <UserLoaderScreen requireLoggedInUser={requireLoggedInUser}>
+      <WrappedComponent {...props} />
+    </UserLoaderScreen>
```

Comment:
Please move the UserLoaderScreen component into this file. There's not a great reason to split these files out -- I think it just obfuscates the code when you have too many files floating around.

Reply:
Fixed